packages/components/src/modal/style.scss

```diff
@@ -61,7 +61,7 @@
 .components-modal__header {
 	box-sizing: border-box;
 	border-bottom: $border-width solid $light-gray-500;
-	padding: 0 $grid-size-large;
+	padding: 0 ($grid-size * 3);
```

Comment:
`($grid-size * 3)` feels a little weird to repeat so many times here. I'm wondering if this should be an additional variable: `$grid-size-extra-large` or something.